corehq/apps/smsbillables/models.py

```diff
@@ -367,3 +399,51 @@ def _get_usage_fee(cls, domain, direction):
                 % (direction, domain)
             )
         return usage_fee
+
+    @classmethod
+    def _get_twilio_charges(cls, backend_message_id, direction, couch_id):
+        def _get_twilio_client():
+            twilio_backend = SQLTwilioBackend.get_global_backends_for_this_class(SQLTwilioBackend.SMS).get()
```

Comment:
we should pass the backend_instance into the _get_twilio_charges method so we can use the actual backend that sent the message:

`twilio_backend = SQLMobileBackend.load(backend_instance, api_id=SQLTwilioBackend.get_api_id(), is_couch_id=True, include_deleted=True)`